src/managers/GuildEmojiRoleManager.js

```diff
@@ -1,18 +1,28 @@
 'use strict';
 
 const Collection = require('../util/Collection');
-const Util = require('../util/Util');
 const { TypeError } = require('../errors');
 
 /**
- * Stores emoji roles
- * @extends {Collection}
+ * Manages API methods for roles belonging to emojis and stores their cache.
  */
-class GuildEmojiRoleStore extends Collection {
+class GuildEmojiRoleManager {
   constructor(emoji) {
-    super();
+    /**
+     * The emoji belonging to this manager.
+     * @type {GuildEmoji}
+     */
     this.emoji = emoji;
+    /**
+     * The Guild belonging to this manager.
```

Comment:
```suggestion
     * The Guild belonging to this manager
```

Also seems weird how emoji and client are not capitalised, but Guild is here